docs/en/latest/how-to-build.md

```diff
@@ -125,8 +125,6 @@ Follow the steps below to install Apache APISIX via the source release package.
   cd apisix-${APISIX_VERSION}
   # Create dependencies
   make deps
-  # Install apisix command
-  make install
```

Comment:
We can also remove `make uninstall`?

Reply:
Sorry for my mistake, I'll change it immediately